src/App.tsx

```diff
@@ -1,14 +1,36 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { useDispatch } from 'react-redux';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { actions } from './features/todos';
+import { Todo } from './types/Todo';
+import { useAppSelector } from './app/hooks';
 
 export const App: React.FC = () => {
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  const dispatch = useDispatch();
+  const setTodos = (todos: Todo []) => dispatch(actions.setTodos(todos));
+
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  useEffect(() => {
+    getTodos().then(result => {
+      setIsLoaded(true);
+      setTodos(result);
+    }).catch(error => {
+      // eslint-disable-next-line no-console
+      console.log(error);
```

Comment:
Remove the console statement. You can just throw error in the catch statement

Reply:
Done